src/bda/plone/shop/at.py

```diff
@@ -130,6 +130,11 @@ def default_item_cart_count_limit(context):
 def default_item_quantity_unit(context):
     return lambda: get_shop_article_settings().default_item_quantity_unit
 
+@implementer(IFieldDefaultProvider)
+@adapter(IBuyable)
+def default_item_minimum_stock(context):
+    return lambda: get_shop_article_settings().default_item_minimum_stock
+
```

Comment:
This is only necessary once you provide some kind of MinimumItemStock AT schema extender and adapter which can override the default value from article settings on dedicated context. You can either remove the lines or complete the implementation ;). Same applies to https://github.com/bluedynamics/bda.plone.shop/pull/63/files#diff-244eaa6a40836e7820844d9d27a6420fR67 just related to DX a behavior